core/lib/zksync_core/src/eth_sender/tests.rs

```diff
@@ -105,20 +105,23 @@ impl EthSenderTester {
             Aggregator::new(
                 aggregator_config.clone(),
                 store_factory.create_store().await,
+                false,
```

Comment:
can you add 1 test with 4844 mode enabled and custom sender? (or is already somewhere and I've missed it?)